server.go

```diff
@@ -243,6 +252,7 @@ func (server *mongoServer) AbendSocket(socket *mongoSocket) {
 		return
 	}
 	server.liveSockets = removeSocket(server.liveSockets, socket)
+	server.activeSocketsCount = len(server.liveSockets)
```

Comment:
Are there any other places where a socket would be released from liveSockets where this value would need to be updated?

Reply:
none, have updated all places.